assets/scss/404.css

```diff
@@ -0,0 +1,168 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300;400&display=swap");
+@-webkit-keyframes swing {
+  0% {
```

Comment:
Modify your live sass compiler so that the project files are as follows:

- assets/css -> This would be generated
- assets/scss -> This is the one to be taken care of  (Only scss files)

[Reference video](https://www.youtube.com/watch?v=_a5j7KoflTs)